kube_apiserver_metrics/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - Kube_apiserver_metrics
 
+## 1.2.0 / 2019-12-02
+
+* [Added] Handle scheme in prometheus_url instead of separate option. See [#4913](https://github.com/DataDog/integrations-core/pull/4913).
```

Comment:
Should we add a note that `scheme` has been deprecated as a result?

```suggestion
* [Added] Handle scheme in `prometheus_url` instead of the separate `scheme` option, which is now deprecated. See [#4913](https://github.com/DataDog/integrations-core/pull/4913).
```